go-controller/pkg/config/config_test.go

```diff
@@ -774,7 +774,7 @@ server-cacert=/path/to/sb-ca.crt`), 0644)
 
 		It("configures server northbound SSL correctly", func() {
 			fakeCmds := ovntest.AddFakeCmd(nil, &ovntest.ExpectedCmd{
-				Cmd: "ovn-nbctl set-connection pssl:6641",
+				Cmd: "ovn-nbctl set-connection pssl:6641 -- set connection . inactivity_probe=0",
```

Comment:
We support ssl. Btwn "config_test.go" is a test.

Reply:
@shettyg In Openshift we don't yet set up the --private-key, --certificate, and --ca-cert that are needed for this. There is planned work to do this. The current code does tcp:<ip>:6641 because of this. Should this use tcp when db access uses tcp?

Reply:
Right. This commit only adds " -- set connection . inactivity_probe=0". If you use TCP or SSL, it does not matter. It will just add the extra string. So it will work for both TCP and SSL. (If you are interested in knowing how to use SSL with ovn-kubernetes, the setup-master.sh and setup-minion.sh in vagrant directory are good examples.